source/automem/traits.d

```diff
@@ -123,3 +123,68 @@ template PointerTarget(T)
     auto s = RefCounted!Point(2, 3);
     static assert(is(Point == PointerTarget!(typeof(s))));
 }
+
+/**
+   Determines if a type `U` is a PointerTarget of `T`
+ */
+template isPointerTargetOf(T, U)
```

Comment:
I don't see how `isPointerTargetOf!(T, U)` is significantly better than `isPointerTarget!T == U` to justify its existence. The template constraints don't even add anything the two individual existing `PointerTarget` templates already have.

Reply:
I have removed that function and just kept the example.